patches/tModLoader/Terraria/Localization/Content/pl-PL/tModLoader.json

```diff
@@ -502,23 +502,23 @@
 		"DiscordLink": "Znajdź pomoc na serwerze Discord",
 		"MSPortToLatest": "Uruchom program tModPorter",
 		"MSFirstLaunchModderWelcomeMessage": "Witaj w menu Tworzenia modów. To menu jest przeznaczone dla osób tworzących mody. Tutaj będziesz mógł budować swój mod.\n\nJeśli jesteś nowy w tworzeniu modów, kliknij poniżej przycisk \"Zobacz na GitHubie\", aby odwiedzić naszą wiki i przejrzeć sekcję \"I am new to modding\".",
-		// "MSNoModSourcesLinkToBasicModdingGuide": "There are no mod sources yet.\n\nIf you are new to modding, please click here to view the \"Basic tModLoader Modding Guide\" which will guide you through your first mod.",
+		"MSNoModSourcesLinkToBasicModdingGuide": "Nie ma jeszcze źródeł modów.\n\nJeśli jesteś nowy w tworzeniu modów, kliknij tutaj, aby zapoznać się z \"Podstawowym przewodnikiem modowania tModLoader\".",
 		"MSNetSDKNotFound": "Tworzenie modów dla tModLoadera wymaga zainstalowania .NET {0} SDK, ale nie zostało to wykryte. Kliknij poniżej przycisk \"Pobierz .NET SDK\" w celu uzyskania instrukcji instalacji. Po zainstalowaniu może być konieczne ponowne uruchomienie tModLoadera.",
 		"DownloadNetSDK": "Pobierz .NET SDK",
 		"DevModsInSandbox": "tModLoader nie może rozwijać modów podczas działania w piaskownicy (np. w przypadku Flatpak). Upewnij się, że zainstalowano Steam bezpośrednio na Twoim systemie.",
 
 		// Create Mod
-		// "CreateModName": "ModName (no spaces)",
-		// "CreateModDisplayName": "Mod DisplayName",
-		// "CreateModAuthor": "Mod Author",
-		// "CreateModBasicSword": "BasicSword (no spaces)",
-		// "CreateModBasicSwordHint": "Leave Blank to Skip",
-		// "CreateModFolderAlreadyExists": "Folder already exists",
-		// "CreateModNameInvalid": "ModName is invalid C# identifier. Remove spaces.",
-		// "CreateModNameReserved": "ModName is a reserved mod name. Choose a different name.",
-		// "CreateModBasicSwordInvalid": "BasicSword is invalid C# identifier. Remove spaces.",
-		// "CreateModDisplayNameEmpty": "DisplayName can't be empty",
-		// "CreateModAuthorEmpty": "Author can't be empty",
+		"CreateModName": "NazwaModa (bez spacji)",
+		"CreateModDisplayName": "WyświetlanaNazwaModa",
+		"CreateModAuthor": "Autor Moda",
+		"CreateModBasicSword": "Podstawowy Miecz (bez spacji)",
```

Comment:
Should there be a space between 'Podstawowy' and 'Miecz,' or should it be written as one word ('PodstawowyMiecz')